src/keyfile.c

```diff
@@ -628,34 +631,44 @@ static void save_ui_prefs(GKeyFile *config)
 	}
 }
 
+typedef enum ConfigPayload
+{
+	PREFERENCES,
```

Comment:
Minor nitpick: Can you please shorten this to PREFS (and above PREFERENCES_FILE -> PREFS_FILE)? We usually refer to the configuration as "prefs" throughout the source and this would follow the pattern.

Reply:
Done in df06733